tests/testthat/test-join_ald_scenario.R

```diff
@@ -221,8 +221,8 @@ test_that("include/excludes `plant_location`s inside/outside a region", {
   region_isos_toy <- tribble(
     ~region, ~isos, ~source,
     "north america", "cn", "demo_2020",
-    "oecd", "de", "demo_2020",
-    "oecd", "fr", "demo_2020",
+    "oecd_europe", "de", "demo_2020",
```

Comment:
Good catch, but I wonder why you chose to add "_europe" here instead of removing it below. That would make this test consistent with other tests and with the dataset `r2dii.data::region_isos`. Any comment?

Reply:
no comment, your suggestion makes more sense!